app/gui/view/welcome-screen/src/side_menu.rs

```diff
@@ -58,4 +88,36 @@ impl SideMenu {
         projects_list.append_or_warn(&button, logger);
         button
     }
+
+    fn setup_new_project_event_listener(&self, create_project: CreateProject) {
+        let closure = Box::new(move |_event: MouseEvent| {
+            create_project.emit(None);
+        });
+        let closure: ClickClosure = Closure::wrap(closure);
+        let callback = closure.as_ref().unchecked_ref();
+        if self.new_project_button.add_event_listener_with_callback("click", callback).is_err() {
+            error!(self.logger, "Could not add create project event listener.");
+        }
+        self.closures.borrow_mut().push(closure);
+    }
+
+    fn setup_open_project_event_listener(&self, entry: &Element, project_name: &str) {
+        let open_project = self.open_project.clone_ref();
+        let project = project_name.to_owned();
+        let closure = Box::new(move |_event: MouseEvent| {
+            open_project.emit(project.clone());
+        });
+        let closure: ClickClosure = Closure::wrap(closure);
+        let callback = closure.as_ref().unchecked_ref();
+        if entry.add_event_listener_with_callback("click", callback).is_err() {
+            error!(self.logger, "Could not add open project event listener for {project_name}.");
+        }
+        self.closures.borrow_mut().push(closure);
```

Comment:
Let's keep the closure reference in the FRP network + create abstract function for gluing DOM callbacks to FRP (as agreed on the call)